roles/setup_pemserver/tasks/setup_pemserver.yml

```diff
@@ -65,6 +65,20 @@
 - name: Update pem probes and alerts
   import_tasks: pem_server_probe_alert.yml
 
+- name: Gather service facts
+  service_facts:
+
+- name: Open PEM TCP port 8443
```

Comment:
Same here. Let's keep the port in a variable

Reply:
We will address this point later.